packages/cerebral/src/Controller.js

```diff
@@ -2,7 +2,7 @@ import DependencyStore from './DependencyStore'
 import FunctionTree from 'function-tree'
 import Module from './Module'
 import Model from './Model'
-import {ensurePath, isDeveloping, throwError, isSerializable, verifyStrictRender, forceSerializable, isObject, getProviders} from './utils'
```

Comment:
isDeveloping could be a constant exposed from utils instead of function. It should do a trick with code elimination and still keep sources readable.